Kernel/style.m

```diff
@@ -170,7 +187,17 @@
   $ruleArrowStyle -> GrayLevel[0.65],
   $ruleGridColor -> GrayLevel[0.85],
   $ruleImageSizePerPlotRange -> 128,
+  
+  (* MultiwaySystem styles *)
+  
+  (* States graph *)
+  $statesGraphVertexStyle -> Directive[Opacity[0.7], Hue[0.62, 0.45, 0.87]],
+  $statesGraphEdgeStyle -> Directive[{Hue[0.75, 0, 0.35]}],
```

Comment:
Sorry for nit, but we don't need a `Directive` if there is only one argument. Same for the two cases below.